### PR TITLE
Update load-generators.md

### PR DESCRIPTION
**Pull request by ValGrace**
Edited documentation

**Notes for Reviewers**

This PR fixes #

grammatical errors in the section

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 371486dd56d5cd1daed4dfb7c2b8404a890ea8df at: https://652bb07a603f7a53f70d1b4a--meshery-docs-preview.netlify.app

File: docs/pages/extensibility/load-generators.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ language: en
 list: include
 ---
 
-Users may prefer to use one load generator over the next given the difference of capabilities between load generators, so Meshery provides a `load generator interface` (a gRPC interface) behind which a load generator can be implemented. Meshery provides users with choice of which load generator they prefer to use for a given performance test. Users may set their configure their own preference of load generator different that the default load generator.
+Users may prefer to use one load generator over the next given the difference of capabilities between load generators, so Meshery provides a `load generator interface` (a gRPC interface) behind which a load generator can be implemented. Meshery provides users with choice of which load generator they prefer to use for a given performance test. Users may configure their own preference for the load generator, different from the default load generator.
```
Comment (leecalcote) [inline docs/pages/extensibility/load-generators.md:12]: Thanks for this, @ValGrace. Unfortunately, this rephrasing takes on a different meaning. We'll either need to leave the current phrasing or think of an alternative.

File: docs/pages/extensibility/load-generators.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ language: en
...
 
 ### What function do load generators in Meshery provide? 
 
-Load generators will provide the capability to run load tests from Meshery. As of today the load generators are embedded as libraries in Meshery and Meshery invokes the load generators APIs with the right load test options to run the load test. At the moment, Meshery has support for HTTP load generators. Support for GRPC and TCP load testing is on the roadmap. Meshery has functional integration with fortio, wrk2, and nighthawk.
+Load generators will provide the capability to run load tests from Meshery. As of today the load generators are embedded as libraries in Meshery and Meshery invokes the load generators APIs with the right load test options to run the load test. At the moment, Meshery only supports HTTP load generators. Support for gRPC and TCP load generators is on the roadmap. Meshery has functional integration with fortio, wrk2, and nighthawk.
```
Comment (leecalcote) [inline docs/pages/extensibility/load-generators.md:16]:
```suggestion
Load generators will provide the capability to run load tests from Meshery. As of today the load generators are embedded as libraries in Meshery and Meshery invokes the load generators APIs with the right load test options to run the load test. At the moment, Meshery supports HTTP load generation. Support for gRPC and TCP load generation is on the roadmap. Meshery has functional integration with fortio, wrk2, and nighthawk.
```

Comment (leecalcote) [reply docs/pages/extensibility/load-generators.md:16]: I'll offer a slight tweak to consider, @ValGrace

File: docs/pages/extensibility/load-generators.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/extensibility/load-generators.md:12]: Users do you have preference settings that they can make to modify how it is that their chosen low generator behaves. And this is something that can be documented. You can use playground.meshery.io to quickly see what preferences people can set. 

However, what is being discussed in this sentence is the fact that users can choose between load generators.

**Comment by l5io** [comment]
🚀 Preview for commit cb90af3e8031be48ff675839919b9f1ee9cbfc8d at: https://652cffa1603f7a1fd90d2163--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Thanks, @ValGrace 👍

**Pull request merged**